doc/sphinx/changes.rst

```diff
@@ -14,7 +14,85 @@ Version 8.15
+
+  - :ref:`Improvements <815Auto>` to the :tacn:`auto` tactic family,
+    fixing the :cmd:`Hint Unfold` behavior, and generalizing the use
+    of discrimination nets.
+
+  - The :tacn:`typeclasses eauto` tactic has a new :ref:`best_effort <815BestEffort>`
+    option allowing it to return *partial* solutions to a proof search problem,
+    depending on the mode declarations associated to each constraint.
+    This mode is used by typeclass resolution during type inference to
+    provide more precise error messages.
+
+  - Many :ref:`commands and options <815Commands>` were deprecated or removed
+    after deprecation and more consistently support locality attributes.
+
+  - The :cmd:`Import` command is extended with :token:`import_categories`
+    to :ref:`select the components <815Import>` of a module to import or not, including
+    features such as hints, coercions, and notations.
+
+  - A :ref:`visual Ltac debugger <815LtacDebugger>` is now available in CoqIDE.
+
+See the `Changes in 8.15.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.15 <https://coq.github.io/doc/v8.15/refman>`_,
+`documentation of the 8.15 standard library <https://coq.github.io/doc/v8.15/stdlib>`_
+and `developer documentation of the 8.15 ML API <https://coq.github.io/doc/v8.15/api>`_
+are also available.
+
+Emilio Jesús Gallego Arias, Gaëtan Gilbert, Michael
+Soegtrop and Théo Zimmermann worked on maintaining and improving the
+continuous integration system and package building infrastructure.
+
+Erik Martin-Dorel has maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+The OPAM repository for Coq packages has been maintained by
+Guillaume Claret, Karl Palmskog, Matthieu Sozeau and Enrico Tassi with
+contributions from many users. A list of packages is available at
+https://coq.inria.fr/opam/www/.
+
+The `Coq Platform <https://github.com/coq/platform>`_ has been maintained
+by Michael Soegtrop and Enrico Tassi.
+
+Our current maintainers are Yves Bertot, Frédéric Besson, Ali Caglayan, Tej
+Chajed, Cyril Cohen, Pierre Corbineau, Pierre Courtieu, Maxime Dénès,
+Jim Fehrle, Julien Forest, Emilio Jesús Gallego Arias, Gaëtan Gilbert,
+Georges Gonthier, Benjamin Grégoire, Jason Gross, Hugo Herbelin,
+Vincent Laporte, Olivier Laurent, Assia Mahboubi, Kenji Maillard,
+Guillaume Melquiond, Pierre-Marie Pédrot, Clément Pit-Claudel, Pierre Roux,
+Kazuhiko Sakaguchi, Vincent Semeria, Michael Soegtrop, Arnaud Spiwack,
+Matthieu Sozeau, Enrico Tassi, Laurent Théry, Anton Trunov, Li-yao Xia
+and Théo Zimmermann. See the `Coq Team face book <https://coq.inria.fr/coq-team.html>`_
+page for more details.
+
+The 41 contributors to this version are
+Tanaka Akira, Frédéric Besson, Juan C, Ali Caglayan, Cyril Cohen, Columbus240, Adrian Dapprich, Maxime Dénès,
```

Comment:
@Columbus240, would you like this to show your real name?  If so, please remind us what it is.

Reply:
Also, I think Juan C is @JuanCoRo aka Juan Conejero.

Reply:
I’m Stéphane Desarzens, thanks for asking.

Reply:
It might be worth fixing the contributors list in the 8.14 release notes as well.

Reply:
Yes, Juan C is Juan Conejero. Thanks for pointing it out :)